gst/nnstreamer/nnstreamer_subplugin.c

```diff
@@ -69,9 +84,32 @@ get_subplugin (subpluginType type, const char *name)
         _spdata_destroy);
 
   table = subplugins[type];
-  data = g_hash_table_lookup (table, name);
 
-  if (data == NULL) {
+  if (searchAlgorithm[type] == NNS_SEARCH_GETALL) {
+    nnsconf_type_path conf_type = (nnsconf_type_path) type;
+    subplugin_info_s info;
+    guint i;
+    guint ret = nnsconf_get_subplugin_info (conf_type, &info);
+
+    for (i = 0; i < ret; i++) {
+      const gchar *fullpath = info.paths[i];
+
+      dlerror ();
```

Comment:
Is it intended?

Reply:
Yes. It is intended to clean up error messages before calling dlopen.

However, because we call dlerror() if dlopen() returns NULL anyway, this is not necessary.